assets/app/view/invite_game.rb

```diff
@@ -8,17 +8,45 @@ class InviteGame < Snabberb::Component
     include GameManager
 
     needs :user
+    needs :refreshing, default: false, store: true
 
     def render
+      game_refresh
+
       children = [h(:h2, 'This game has not started yet')]
       if @user
-        children << h(:div, [h(GameCard, gdata: @game, user: @user)])
+        children << h(:div, [h(GameCard, gdata: @game_data, user: @user)])
       else
         children << h(:h3, 'You need to login before joining a game:')
         children << h(View::User, type: :login)
       end
 
-      h('div', children)
+      destroy = lambda do
+        `clearTimeout(#{@refreshing})`
+        store(:refreshing, nil, skip: true)
```

Comment:
do you need store(:refreshing) here? i'm curious if it's needed -- not sure

Reply:
I think the settimeout function returns an id and not the object itself, but I didn't check. This is just copy-pasted from Home